app/src/main/java/org/gdg/frisbee/android/event/EventActivity.java

```diff
@@ -110,13 +98,12 @@ public int getItemPosition(Object object) {
 
         @Override
         public int getCount() {
-            int count = mContext.getResources().getStringArray(R.array.event_tabs).length;
```

Comment:
Yep I know :) I was also trying to mess with you. 
What are we doing? It is Friday night 1AM here. :)